python/oneflow/framework/docstr/tensor.py

```diff
@@ -169,9 +169,121 @@
 add_docstr(
     oneflow.Tensor.to_global,
     """
-    Tensor.to_global() -> Tensor
+    Tensor.to_global(placement=None, sbp=None, grad_sbp=None) -> Tensor
 
-    See :func:`oneflow.to_global`
+    Creates a global tensor if this tensor is a local tensor, otherwise performs Tensor placement and/or sbp conversion.
+
+    Note:
+        This tensor can be local tensor or global tensor.
+
+        - For local tensor
+
+          Both placement and sbp are required.
+
+          The returned global tensor takes this tensor as its local component in the current rank.
+
+          There is no data communication usually, but when sbp is ``oneflow.sbp.broadcast``, the data on rank 0 will be broadcast to other ranks.
+
+        - For global tensor
+
+          At least one of placement and sbp is required.
+
+          If placement and sbp are all the same as this tensor's own placement and sbp, then returns this tensor own.
+    
+    Args:
+        placement (flow.placement, optional): the desired placement of returned global tensor. Default: None
+        sbp (flow.sbp.sbp or tuple of flow.sbp.sbp, optional): the desired sbp of returned global tensor. Default: None
+        grad_sbp (flow.sbp.sbp or tuple of flow.sbp.sbp, optional): manually specify the gradient sbp of this operation in the backward pass. If None, the gradient sbp will be infered automatically. It is only used if this tensor is a global tensor. Default: None
```

Comment:
specify the sbp of this tensor's grad tensor in the backward pass ?

Reply:
done